apis/apnscp_api.php

```diff
@@ -62,6 +62,11 @@ public function apiRequest($function, array $params = [], $session_id = null)
         // Create SOAP connection
         ini_set('default_socket_timeout', 5000);
 
+	$headers = [
+            'Abort-On: error',
+            'X-Forwarded-For: ' . $this->getFromContainer('requestor')->ip_address
```

Comment:
Testing is our friend.  This method is only defined in classes the use the Container trait.  So at the top of the file add
use Blesta\Core\Util\Common\Traits\Container;
And at the top of the class add
    use Container;

There are many classes in Blesta that already load this trait (models, controllers, and classes that extend Plugin, Module, and Gateway) but this file is not one of them.

Reply:
> Testing is our friend. 

Mea culpa. Updated.